packages/create-block/lib/init-wp-scripts.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+const { command } = require( 'execa' );
+const { install } = require( 'pkg-install' );
+const { join } = require( 'path' );
+const writePkg = require( 'write-pkg' );
+
+/**
+ * Internal dependencies
+ */
+const { info } = require( './log' );
+
+module.exports = async function( { author, license, slug, title, version } ) {
+	const cwd = join( process.cwd(), slug );
+
+	info( '' );
+	info( 'Creating a "package.json" file.' );
+	await writePkg( cwd, {
+		name: slug,
+		version,
+		description: title,
```

Comment:
If we prompt for a description, is there a reason we shouldn't want to use it here?

Reply:
True, we might use it instead 👍

Reply:
Done 560492c88483a28025920f87781026b8df6d918c